tests/conftest.py

```diff
@@ -213,42 +194,6 @@ def pytest_runtest_teardown(item, nextitem):
     reset_logging(logger_name=None)  # Reset the root logger as well
 
 
-# This fixture will be used by all tests.
-@pytest.fixture(scope="function", autouse=True)
-def _set_use_cpp(request: pytest.FixtureRequest):
-
-    do_use_cpp: bool = True
-
-    # Check for the param if this was indirectly set
-    if (hasattr(request, "param") and isinstance(request.param, bool)):
-        do_use_cpp = request.param
-    else:
-        # If not, check for the marker and use that
-        use_cpp = request.node.get_closest_marker("use_cpp") is not None
-        use_python = request.node.get_closest_marker("use_python") is not None
-
-        if (use_cpp and use_python):
-            raise RuntimeError(f"Both markers (use_cpp and use_python) were added to function {request.node.nodeid}. "
-                               "Remove markers to support both.")
-
-        # This will default to True or follow use_cpp
-        do_use_cpp = not use_python
-
-    from morpheus.config import CppConfig
-
-    CppConfig.set_should_use_cpp(do_use_cpp)
-
-    yield do_use_cpp
-
-
-# This fixture will be used by all tests.
-@pytest.fixture(scope="function")
-def use_cpp(_set_use_cpp: bool):
-
-    # Just return the set value
-    yield _set_use_cpp
-
```

Comment:
Below this, the fixtures `config_only_cpp` and `config_no_cpp` were unchanged. Should these be updated too?

Reply:
good catch